tests/driver/driver.ts

```diff
@@ -0,0 +1,296 @@
+   */
+  copyFileFromRoot(relPath: string) {
+    const sandboxPath = this.getSandbox();
+    const sourcePath = path.resolve(REPO_ROOT, relPath);
+    const destPath = path.resolve(sandboxPath, relPath);
+    const destDir = path.parse(destPath).dir;
+    fs.mkdirSync(destDir, { recursive: true });
+    fs.copyFileSync(sourcePath, destPath);
+  }
+
+  /**
+   * Moves/renames a file at from the `sourceRelPath` inside the sandbox to the `destRelPath`.
+   * Recursively creates the destination's parent directory if it does not exist.
+   */
+  moveFile(sourceRelPath: string, destRelPath: string) {
+    const sandboxPath = this.getSandbox();
+    const sourcePath = path.resolve(sandboxPath, sourceRelPath);
+    const destPath = path.resolve(sandboxPath, destRelPath);
+    const destDir = path.parse(destPath).dir;
+    fs.mkdirSync(destDir, { recursive: true });
+    fs.renameSync(sourcePath, destPath);
+  }
+
+  /**
+   * Copies a file at from the `sourceRelPath` inside the sandbox to the `destRelPath`.
+   * Recursively creates the destination's parent directory if it does not exist.
+   */
+  copyFile(sourceRelPath: string, destRelPath: string) {
+    const sandboxPath = this.getSandbox();
+    const sourcePath = path.resolve(sandboxPath, sourceRelPath);
+    const destPath = path.resolve(sandboxPath, destRelPath);
+    const destDir = path.parse(destPath).dir;
+    fs.mkdirSync(destDir, { recursive: true });
+    fs.cpSync(sourcePath, destPath);
+  }
+
+  /**
+   * Deletes a file at the `relPath` inside the sandbox root.
+   */
+  deleteFile(relPath: string) {
+    const sandboxPath = this.getSandbox();
+    const targetPath = path.resolve(sandboxPath, relPath);
+    fs.rmSync(targetPath, { recursive: true });
+  }
+
+  /**** Trunk config methods ****/
+
+  /**
+   * Return the yaml result of parsing the sandbox's .trunk/trunk.yaml file.
+   */
+  getTrunkConfig = (): any => {
+    if (this.sandboxPath) {
+      return getTrunkConfig(this.sandboxPath);
+    }
+  };
+
+  /**
+   * Return the yaml result of parsing the output of `trunk config print` in the test sandbox.
+   */
+  getFullTrunkConfig = (): any => {
+    const printConfig = execSync(`${ARGS.cliPath ?? "trunk"} config print`, {
+      cwd: this.sandboxPath,
+    });
+    return YAML.parse(printConfig.toString());
+  };
+
+  async runTrunkCmd(
+    argStr: string,
+    execOptions?: ExecOptions
+  ): Promise<{ stdout: string; stderr: string }> {
+    return await this.runTrunk(argStr.split(" ").filter(Boolean), execOptions);
+  }
+
+  /**
+   * Run a specified trunk command with `args` and additional options.
+   * @param args arguments to run, excluding `trunk`
+   * @param execOptions
+   */
+  async runTrunk(
+    args: string[],
+    execOptions?: ExecOptions
+  ): Promise<{ stdout: string; stderr: string }> {
+    const trunkPath = ARGS.cliPath ?? "trunk";
+    return await execFilePromise(trunkPath, args.filter(Boolean), {
```

Comment:
nit: potentially not ok to use Boolean here, javascript does weird things imo, e.g. if you pass in an input "0" or "false" as part of your array

Reply:
replaced